NEWS

```diff
@@ -7,13 +7,183 @@
 ##
 #
 
-## Current
+## 1.12.0 RC1 (December 2, 2021)
 ### Features:
+#### Core
+* Added beta-level support for Go language bindings
+* Added new objects to VFS (md, component, log_level, etc)
+* Added configuration variable to specify which loadable modules are allowed
+* Added build-time configuration to disable sigaction overriding
 #### UCP
-* Added API for querying UCP library attributes
+* Added address versioning to correctly preserve wire compatibility since v1.11.0
+* Added new client/server connection establishment packet header format
+* Enabled rendezvous and tag sync protocols when error handling is enabled on the endpoint
+* Added iov zcopy support to RMA operations
+* Reduced memory usage of unexpected messages by fitting receive buffer size to packet size
+* Added support for modifying UCT and UCS configs by ucp_config_modify() API
+* Optimized unpacked rkeys memory consumption
+* Added request flag to influence latency vs. bandwidth protocol
+* Reduced memory management overhead with new protocols
+* Improved performance calculations for new protocols
+* Added AMO support with GPU memory target using new protocols
+* Added put_zcopy, get_zcopy and pipeline based rendezvous in new protocols
+* Added support for user-defined alignment in Active Messages
+* Added support for offload tag sync in new protocols
+* Updated ucp_atomic_post() to use NBX flow
+##### API
```

Comment:
but this is UCP API changes, it has nested level (extra `#`)

Reply:
or you mean move it right after `####UCP`?

Reply:
Yes.

Reply:
moved to the top, but removed `#####API` line, because:
1. We do not have such section for other parts (UCT, UCS), so it is consistent with other parts - that API changes are just on the top
2. Otherwise would need to introduce some other caption which would separate API and plain features